packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -170,7 +170,7 @@ function CustomDrawerContent(props: DrawerContentComponentProps<DrawerContentOpt
       <View style={styles.drawerBottom}>
         <Text style={fontStyles.label}>{t('address')}</Text>
         <AccountNumber address={account || ''} location={Screens.DrawerNavigator} />
-        <Text style={styles.smallLabel}>{`Version ${appVersion}`}</Text>
+        <Text style={styles.smallLabel}>{`${t('version')} ${appVersion}`}</Text>
```

Comment:
We should never concatenate translatable strings as it will make it impossible to translate correctly in some other languages where words ordering is different.
```suggestion
        <Text style={styles.smallLabel}>{t('version', { appVersion })}</Text>
```